docs/guidance/rayjob.md

```diff
@@ -0,0 +1,129 @@
+## Ray Job (alpha)
```

Comment:
We should explain what the operational difference is between a Ray Service and a Ray Job.

Reply:
Add some details. The cluster can be deleted by RayJob controller once it succeed or failed